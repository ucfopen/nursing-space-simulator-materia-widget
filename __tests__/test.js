import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CameraFP from '../src/js/components/assets/camera_fp';

describe('Loads Components', () => {
	it('Camera renders aframe Camera', () => {
		const component = renderer.create(<CameraFP />);
		const json = component.toJSON();
		expect(json).toMatchSnapshot();
	});
})