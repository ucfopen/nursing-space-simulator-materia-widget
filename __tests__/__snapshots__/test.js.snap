// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loads Components Camera renders aframe Camera 1`] = `<a-entity />`;
